calendar_event_type_color/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module allows to set color on event type in tree view.
```

Comment:
The field is not stored, so it cannot be set, it just takes the color of the first category (not the first inserted, the first according to the order of the categories).

Reply:
Actually the field is already there in core, but this module does expose it on the tree view